app/src/db/models/tables/objectModel.js

```diff
@@ -78,6 +78,22 @@ class ObjectModel extends Timestamps(Model) {
       filterActive(query, value) {
         if (value !== undefined) query.where('object.active', value);
       },
+      filterVersionAttributes(query, mimeType, deleteMarker, isLatest) {
```

Comment:
might be simpler to move lines 83 - 95 to the objectService() function and not put them in a modifier at all